pkgs/development/libraries/x264/default.nix

```diff
@@ -34,6 +34,10 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "lib" "dev" ];
 
+  STRINGS = if stdenv.hostPlatform.useLLVM or false
```

Comment:
Hm, smells like we would need a `stdenv.bintools.isLLVM or false` of some kind…

Reply:
I guess LLVM bintools and GCC could be mixed, but would anyone actually do that?